examples/01_dirty_categories.py

```diff
@@ -1,274 +1,289 @@
 """
-Dirty categories: machine learning with non normalized strings
-==============================================================
+dirty_cat: Tackling Non-Normalized Categories In Machine Learning Settings
+==========================================================================
 
-Including strings that represent categories often calls for much data
-preparation. In particular categories may appear with many morphological
-variants, when they have been manually input or assembled from diverse
-sources.
+Handling strings that represent categories often requires substantial data
+preparation, especially when they contain various morphological variants
+emanating from manual input or diverse data sources.
 
-Here we look at a dataset on wages [#]_ where the column 'Employee
-Position Title' contains dirty categories. On such a column, standard
-categorical encodings leads to very high dimensions and can lose
-information on which categories are similar.
+..note:
+    A variable is considered categorical when it can take on one of a
+    limited, and usually fixed (discrete) number of possible values.
+    It is opposed to continuous variables and free text,
+    which can take an infinite number of unique values.
```

Comment:
```suggestion
..note:
    A variable is categorical when it takes a
    fixed and limited number of possible values, 
    as opposed to continuous variables and free text,
    which can take large or infinite number of unique values.
```
Trying to make the note shorter (as this is just on the side)